### ISSUE TITLE
Omitting "number-part" of signatures leads to collisions

### ISSUE THREAD

**Issue by MorpheusXAUT** [exodus4d/pathfinder#419]
I've just ran into a problem with a signature collision in pathfinder's sig tracker caused by the omission of the "number-part" of sig IDs.

![Demo screenshot](http://i.imgur.com/LSDQbgT.png)

As you can see on the screenshot above, there is currently two signatures with the same "letter-part" of the sig ID, but different "number-parts". Since pathfinder completely ignores the numbers in a sig ID, this leads to a collision in the signature table and thus doesn't allow for tracking of both signatures simultaneously.
I'm actually quite surprised to see this as I've never encountered this before. I do prefer the shorter version of sig IDs being displayed, but having all sigs on the tracker is even more useful. 

I guess you could just track the "number-part" internally (for differentiation) and only display the letters in the list? Or add a toggle for showing both if needed...

**Comment by WildStrawberryEVE** [comment]
I found that it omits the number part when you are copy&pasting it through a "signature reader" window but - as a workaround - you can still add such signatures manually, with a number part and they are distinguished.

**Comment by MorpheusXAUT** [comment]
@WildStrawberryEVE Oh, great idea for a workaround, didn't even think about that.

On the long run, having pathfinder either include the numbers for every signature or just for the ones that create collisions would probably be a good solution.

**Comment by exodus4d** [comment]
- I removed the "number stripping" from the name. -> Sig names are now stored with full name in DB.
- Clients will still see the "minified" name on the table (as now)
- I added a new tooltip to the colum, which will show you the full name of a signature:
![sig_tooltip](https://cloud.githubusercontent.com/assets/9299033/21594665/83a466f6-d125-11e6-867e-788ee63a3081.png)
- Signatures with same letters will no longer be overwritten when using the sig-reader.
